modules/rest-api/pages/rest-xdcr-identify-target-cluster.adoc

```diff
@@ -0,0 +1,180 @@
+= Identifying the Target in Replication
+:description: pass:q[]
+
+[abstract]
+{description}
+
+== HTTP Method and URI
+
+----
+GET /xdcr/sourceClusters
+----
+
+== Description
+
+When on a current cluster, XDCR lets you:
+
+* Retrieve the list of remote clusters that replicate to the current cluster.
+If XDCR finds a list of replicating source clusters, it makes the current cluster the target cluster.
```

Comment:
Remove the sentence on line 18 -- "If XDCR finds a list of replicating source clusters, it makes the current cluster the target cluster."    
I don't know what the sentence is trying to say.  The customer is on the target cluster running `GET /xdcr/sourceClusters` to list the incoming replications info that's been sent by the source clusters to this cluster, the target cluster.